app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -58,16 +58,54 @@ export class ModuleMenuCategory extends React.PureComponent<MenuCategoryProps, M
                     {category.name}
                 </div>
                 {isExpanded && category.modules.map((m) => (
-                    <ModuleMenuItem key={m.id} module={m} onSelect={onItemSelect} />
+                    <ModuleMenuItem key={m.id} module={m} />
                 ))}
             </React.Fragment>
         )
     }
 }
 
-const ModuleMenuItem = ({ module, onSelect }) => (
-    /* eslint-disable-next-line */
-    <div className={styles.ModuleItem} role="option" onClick={() => onSelect(module.id)}>
+const onDragStart = (e: any, moduleId: number, moduleName: string, streamId?: string) => {
+    e.stopPropagation()
+    const dragImage = document.querySelector('#dragImage')
```

Comment:
One problem with hard-coded ids is that it potentially causes issues if we ever need to have multiple canvases visible. You could use a ref to capture the element and pass it in